libs/interpreter-lib/src/validation-checks/runtime-parameter-literal.ts

```diff
@@ -75,7 +79,16 @@ function checkRuntimeParameterValueParsing(
     return;
   }
 
-  const metaInf = getMetaInformation(type);
+  let metaInf: MetaInformation | undefined;
+  if (
+    isReferenceableBlocktypeDefinition(type.ref) &&
+    BlockMetaInformation.canBeWrapped(type.ref)
+  ) {
+    metaInf = new BlockMetaInformation(type.ref);
+  } else if (isBuiltinConstrainttypeDefinition(type.ref)) {
+    metaInf = getConstraintMetaInf(type.ref);
+  }
+
```

Comment:
This code snippet occurs at different places. Consider adding a utility function.